bin/yaml/android-java/fetch_art_release_from_head.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Fetches ART binaries and ART bootclasspath jars from AOSP head.
+# Saves the result at `art_release.zip`.
+
+set -e
```

Comment:
```suggestion
set -euo pipefail
```

(if this is not just a copy paste...)

Reply:
This is a copy-paste, but the added flags work. Done!